### DISCUSSION TITLE
Sound Output OPL2 Method for COM

### DISCUSSION THREAD

**Discussion by pioneerman2k1** [86Box/86Box#4053] [Ideas]
Hi Folks,

im new to this so deal severely with me.

Is it possible to implement the Output Feature for OPL2 Cards, such as SB2, to Output the Data Stream via Serial?
I want to use my OPL2 Sound Board:

https://github.com/DhrBaksteen/ArduinoOPL2

https://www.tindie.com/products/cheerful/opl2-audio-board/

It contains the SerialPassthrough.ino for Arduino and catches Data via Serial and Plays it on an original OPL2 Sound Chip.

There is also a DosBox Version aviable, that is Patched to work with the SoundBoard

https://github.com/DhrBaksteen/ArduinoOPL2.DosBox

Hope to hear from you soon.
Thanks in advance
Chris

**Comment by dhrdlicka** [comment]
Hello, serial and parallel passthrough is planned for a future release.

**Comment by pioneerman2k1** [comment]
> Hello, serial and parallel passthrough is planned for a future release.

Nice to hear, but future.. is there somesting precise?

**Comment by OBattler** [comment]
Maybe for version 3.1 when I get around to implementing serial passthrough.

**Comment by binarymaster** [comment]
> Maybe for version 3.1 when I get around to implementing serial passthrough.

There is existing ticket #256 about it, by the way.

You probably can kill two birds with one stone, as #257 will be then fixed by using [com0com](http://com0com.sourceforge.net/) along with PuTTY, for example.

**Comment by Cacodemon345** [comment]
Serial passthrough is now available.
